app/scripts/store/dimStores.directive.js

```diff
@@ -22,7 +22,7 @@
         "    'hide-filtered': vm.hideFilteredItems,",
         "    'show-elements': vm.showElements",
         '  }">',
-        '  <div dim-store-heading store-data="store"></div>',
+        '  <div dim-store-heading class="character dim-col-{{vm.charCol}}" store-data="store"></div>',
```

Comment:
Good points, all. I'll post some fixes when I have some home time (likely
Sunday). I tried stretching the header to full vault width and the loadout
menu looks ridiculous.

On Wednesday, May 11, 2016, kyle shay <notifications@github.com> wrote:

> In app/scripts/store/dimStores.directive.js
> <https://github.com/DestinyItemManager/DIM/pull/503#discussion_r62963375>:
>
> > @@ -22,7 +22,7 @@
> >          "    'hide-filtered': vm.hideFilteredItems,",
> >          "    'show-elements': vm.showElements",
> >          '  }">',
> > -        '  <div dim-store-heading store-data="store"></div>',
> > +        '  <div dim-store-heading class="character dim-col-{{vm.charCol}}" store-data="store"></div>',
>
> hmm.. good call
>
> [image: screen shot 2016-05-12 at 12 19 11 am]
> <https://cloud.githubusercontent.com/assets/424158/15204026/2fa2fb42-17d7-11e6-91aa-b2b3d1cf981a.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/DestinyItemManager/DIM/pull/503/files/068cbf519a76174dc4f12d460b53e577340eda91#r62963375>
>